nixos/tests/podman-apparmor.nix

```diff
@@ -0,0 +1,63 @@
+# This test runs podman and checks AppArmor default confinement
```

Comment:
Need to link this test in a couple of places.

https://github.com/NixOS/nixpkgs/blob/63d559d0056f791c1cb4abd4586af9f9ba53dae2/nixos/tests/all-tests.nix#L369-L371

https://github.com/NixOS/nixpkgs/blob/352a3d75387609469fa23b0bcaaba03541e0a396/pkgs/applications/virtualization/podman/default.nix#L82-L83